bigbluebutton-html5/imports/ui/components/video-provider/video-button/component.jsx

```diff
@@ -62,7 +63,7 @@ const JoinVideoButton = ({
     } else {
       mountVideoPreview();
     }
-  };
+  }, 500);
```

Comment:
Please make the debounce time a constant.

Reply:
done!